job_spec/config/compute_environments.yml

```diff
@@ -1,9 +1,33 @@
+# Compute Environments that are shared between multiple jobs and/or steps should be defined here, 
```

Comment:
This is a bit out-of-place here, but I think its valuable. We can remove it whenever we properly document job spec syntax.

Reply:
I like it for now. Yes, as I was writing the Changelog entry for https://github.com/ASFHyP3/hyp3/pull/2463 I was thinking we just need a formal specification of our job spec language.